src/posts/react-image-gallery.md

```diff
@@ -0,0 +1,211 @@
+---
+title: 'How to use React Image Gallery to build a beautiful and responsive image gallery'
+date: '2022-09-06'
+tags: ['javascript', 'react', 'image-gallery']
+thumbnail: ../images/thumbnails/image-gallery-icon.png
+author: 'Ajeet Chaulagain'
+---
+
+import { QuickInfo } from "../components/utilities/quick-info.tsx"
+
+Suppose you are creating apps or web pages. In that case, you might need an image gallery to share the multiple images on our webpage with the best user interactions possible for the targeted audience. However, creating an image library from scratch that provides a great user experience for to end user requires quite an effort. In this tutorial, we will learn how to use different* APIs of the react-image-gallery* npm package to create a _beautiful and elegant image gallery_ for your need with minimal effort.
+
+_Screenshot of final application:_
+
+![Final application screenshot](../images/react-image-gallery/final-application-screenshot.png)
+
+Here is the final source code and demo of the app:
+
+- [View Source code on Github](https://github.com/ajeetchaulagain/react-image-gallery-demo)
+- [View project demo](https://react-image-gallery-demo.ajeetchaulagain.com/)
+
+## Why react-image-gallery ?
+
+React image gallery is one of the most popular and community-loved npm packages for providing react components to _build highly customizable_ image galleries. Its popularity among consumers is due to its support for a wide range of features out of the box. These features include _swipe gestures for mobile devices, navigation through thumbnails, responsive design, and much more_. Moreover, the react component provided by the package is highly customizable through different props that will help almost all your job get done as a need of gallery in your project. In this tutorial, we will demonstrate the most used props that will be handy for our project.
```

Comment:
back most popular with a link to npm trends like this one: https://npmtrends.com/react-gallery-swiper-vs-react-grid-gallery-vs-react-image-gallery-vs-react-image-lightbox-vs-react-image-slider-vs-react-images

Reply:
Link features to react image gallery feature page or feature on github page.